deploy/infra/aws/terraform/cloud-native/cluster-infra/input.tf

```diff
@@ -157,3 +157,50 @@ variable "create_ssh_rsa_key_pair" {
   description = "If true, this module will create ssh_rsa key pair in the AWS account. This pair can be used during Cloud-Pipeline deployment process as a ssh key for worker nodes."
 }
 
+###################################################################
+#                  AWS RDS for Cloud-Pipeline deployment
+###################################################################
+variable "rds_instance_type" {
+  type        = string
+  default     = "db.m6i.large"
+  description = "The instance type of the RDS instance"
+}
+
+variable "rds_storage_size" {
+  type        = number
+  default     = 200
+  description = "The allocated RDS storage size in gigabytes"
+}
+
+variable "rds_default_db_name" {
+  type        = string
+  default     = "postgres"
+  description = "The DB name to create. If omitted, no database is created initially"
+}
+
+variable "rds_root_username" {
+  type        = string
+  default     = "postgres"
+  description = "Username for the master DB user"
+}
+
+variable "rds_root_password" {
+  type        = string
+  default     = null
+  description = "Password for the default master DB user"
+}
+
+variable "cloud_pipeline_db_configuration" {
+  type = list(object({
+    username = string
+    password = string
+    database = string
+  }))
+  description = "Login and password for database owner in RDS Postgres instance"
+}
+
+variable "rds_default_db_port" {
```

Comment:
it is a port of whole db instance, not only a default database, so -> `rds_db_port`